CampusTour/Helpers/Filter.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  Filter.swift
+//  CampusTour
+//
+//  Created by Ji Hwan Seung on 3/21/18.
+//  Copyright © 2018 cuappdev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+enum Filter: String {
+    case Sort = "Sort"
+    case College = "College"
+    case Food = "Food"
+    case Event = "Event"
+    case Tour = "Tour"
+}
+
+fileprivate let filters: [Filter] = [
+    .Sort,
+    .College,
+    .Food,
+    .Event,
+    .Tour,
+]
+
+class FilterBar: UIView {
+    
+    var scrollView: UIScrollView!
+    var selectedFilters = [Filter]()
+    private var buttons = [UIButton]()
+    private let padding: CGFloat = 8.0
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        scrollView = UIScrollView()
+        scrollView.backgroundColor = Colors.offwhite
+        scrollView.alwaysBounceHorizontal = true
+        scrollView.showsHorizontalScrollIndicator = false
+    }
+    
+    override func layoutSubviews() {
+        addSubview(scrollView)
```

Comment:
might also want to delete the views you're adding in here since if this gets called several times by iOS it will make double views. You could also do this in `init`